tests/foreman/api/test_syncplan.py

```diff
@@ -333,7 +333,7 @@ def test_positive_update_name(self):
                 sync_plan.name = name
                 self.assertEqual(sync_plan.update(['name']).name, name)
 
-    @tier1
+    @tier2
```

Comment:
and why that test case is tier2? Tier2 always means integration testing which means combination of two or more different application components. Here you have one isolated component without any interaction with another component, so it is tier1 for sure

Reply:
> Tier2 always means integration testing

No. Tiers are not just another name for test levels. Equating tier with level was never the intention and is one of misconceptions that we have in Satellite QE. If you want to learn more, I elaborate in [blog post](https://url.corp.redhat.com/dd08eaa).

This change is aligned with improved tier definitions that are currently being discussed. See thread on satqe-list for details.

Reply:
that was a concept that became a standard for many years. And now you want change it just like that? So if that is being discussed, I NACK that PR till we have completely approved approach, not just some blog

Reply:
@mirzal so who decide whether we have `Critical component level functional tests.` or `Non-critical component level functional tests`? Test levels were at least unambiguous, but new criteria is too subjective.

Reply:
`CaseImportance` is decided by component owner. If you see importance value that you strongly disagree with, reach out to component owner and see if you can find some common ground.

Please note that google doc contains non-exhaustive list of metadata, i.e. it provides new guidelines for some fields, but doesn't depreciate any existing field. If `CaseLevel` was unambiguous in the past, it is unambiguous right now - because there wasn't any change in that regard.